### PR TITLE
Add support for embedded album art for audio streams

### PR DESCRIPTION
**Pull request by wdehoog**
See issue #9

### CODE REVIEW

File: src/core/media/gstreamer/engine.cpp [hunk 1]
```diff
@@ -207,14 +208,21 @@ struct gstreamer::Engine::Private
...
+    }
+
     void on_tag_available(const gstreamer::Bus::Message::Detail::Tag& tag)
     {
         media::Track::MetaData md;
 
         // We update instead of creating from scratch if same uri
         auto &tuple = track_meta_data.get();
-        if (playbin.uri() == std::get<0>(tuple))
+        if (playbin.uri() == std::get<0>(tuple))
```
Comment (Flohack74) [inline src/core/media/gstreamer/engine.cpp]: Remove whitspace change.

**Review by Flohack74 — CHANGES_REQUESTED** [review]

File: src/core/media/gstreamer/engine.cpp (hunk 1 above)
Comment (dobey) [reply src/core/media/gstreamer/engine.cpp]: This one tiny whitespace change is perfectly fine in here. No need to block on that alone.

**Review by dobey — CHANGES_REQUESTED** [review]
Can you please add instructions for how this can be tested, to the description? Also, unit tests for the changes would be quite nice. Thanks!

**Comment by UniversalSuperBox** [comment]
Hi @wdehoog, thanks for your contribution.

We have moved the media-hub repository to GitLab. These changes will need to be resubmitted if they are still relevant.

You can find the new copy of this repository here: https://gitlab.com/ubports/core/media-hub

**Pull request closed without merging**